arbos/programs/api.go

```diff
@@ -231,6 +233,37 @@ func newApiClosures(
 		if readOnly {
 			return vm.ErrWriteProtection
 		}
+
+		// Calculate multi-gas cost in a way similar to makeGasLog from gas_table.go for EVM
+		topicBytes := uint64(32)
```

Comment:
I would suggest moving these calculations to `operations_acl_arbitrum`, but I'm not sure because unlike other operations, log gas (ink) is calculated on the wasm side [here](https://github.com/OffchainLabs/nitro/blob/6056c3618d979199e706858b7673ee0699c92f4f/arbitrator/wasm-libraries/user-host-trait/src/lib.rs#L609).

Reply:
I agree with your comment; you should move this logic to operations_acl_arbitrum. This will make this code clearer.